PyInstaller/hooks/hook-keyring.py

```diff
@@ -9,8 +9,10 @@
 # SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
 #-----------------------------------------------------------------------------
 
-# Tested with keyring 3.7 on MacOS.
-
-from PyInstaller.utils.hooks import collect_submodules
+from PyInstaller.utils.hooks import collect_submodules, copy_metadata
 
+# Collect backends
 hiddenimports = collect_submodules('keyring.backends')
+
+# Copy metadata (required for proper backend discovery)
```

Comment:
Please briefly explain, why this is required.
(From the bug-report I assume for finding the backends)

Reply:
OK, I've expanded the comment explaining why the metadata needs to be copied.